CONTRIBUTING.md

```diff
@@ -163,10 +163,179 @@ export default wrapper((props: any) => (
 ), 'foobar')
 ```
 
+### Adding a new Core Widget
```

Comment:
This seems to be duplicate to line 55

Reply:
thanks for the catch. Should be good now.